docs/application/dotnet/guides/multimedia/audio.md

```diff
@@ -137,11 +137,51 @@ To query audio device information:
         ```
 
 
-        > **Note**   
-		> The initial running state of the connected device is `false`, which means the connected device is not running.
+        > **Note**
+        >
+        > The initial running state of the connected device is `false`, which means the connected device is not running.
+
+-   To utilize additional functionality for UsbAudio type and Output direction device, use the [Tizen.Multimedia.AudioDevice](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.AudioDevice.html) class:
```

Comment:
Query:

As per this hyperlink: https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.AudioDeviceType.html#Tizen_Multimedia_AudioDeviceType_UsbAudio

Should UsbAudio be in code tag like `UsbAudio` as it is a field for the ENUM value. Kindly share your opinion regarding the same.

Reply:
@rishu-samsung In this sentence as the "UsbAudio " is used a simple word to describe, not ENUM, I think we don't need to apply the code tag.
For more improvement, if possible, it would be better to change to "USB audio".

Reply:
@mijin85cho Agreed with your point. We can use "USB audio" instead of "UsbAudio".

@sangchul1011 Kindly change "UsbAuido" to "USB audio".

Reply:
It is updated now. Thanks.